kaldb/src/test/java/com/slack/kaldb/chunk/IndexingChunkImplTest.java

```diff
@@ -532,7 +532,7 @@ public void testSnapshotToS3UsingChunkApi() throws Exception {
       assertThat(chunk.snapshotToS3(bucket, "", s3BlobFs)).isTrue();
       assertThat(chunk.info().getSnapshotPath()).isNotEmpty();
 
-      assertThat(getCount(INDEX_FILES_UPLOAD, registry)).isEqualTo(15);
+      assertThat(getCount(INDEX_FILES_UPLOAD, registry)).isEqualTo(18);
```

Comment:
seems super brittle! I just "fixed" it by changing to the new file number count that lucene 9 will produce